src/pretalx/orga/forms/schedule.py

```diff
@@ -1,10 +1,11 @@
 from django.forms import BooleanField, ModelForm, ValidationError
 from django.utils.translation import gettext_lazy as _
+from i18nfield.forms import I18nModelForm
 
 from pretalx.schedule.models import Schedule
 
 
-class ScheduleReleaseForm(ModelForm):
+class ScheduleReleaseForm(ModelForm, I18nModelForm):
```

Comment:
``I18nModelForm`` subclasses ``ModelForm``, so we only need ``I18nModelForm`` here.